Code/Tools/TestImpactFramework/Frontend/Console/Python/Code/Source/TestImpactPythonCommandLineOptions.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <TestImpactPythonCommandLineOptions.h>
+#include <TestImpactCommandLineOptionsUtils.h>
+#include <TestImpactFramework/TestImpactUtils.h>
+
+#include <AzCore/Settings/CommandLine.h>
+
+namespace TestImpact
+{
+    namespace
+    {
+    } // namespace
+} // namespace TestImpact
```

Comment:
question: Can this file just be removed? Nothing going on in here it seems

Reply:
Will remove now! There was some logic here but it got refactored out and I forgot to remove this.

Reply:
Thanks!